bin/oneoff/import_unit_details.rb

```diff
@@ -38,9 +38,9 @@ def parse_options
 
         Usage: #{$0} [options]
 
-        Example: runner.rb -u coursea-2021
-        Example: runner.rb -l -u csd1-2021
-        Example: runner.rb -l -u csp2-2021,csp3-2021,csp4-2021
+        Example: runner.rb -u coursea-2021 -m LessonGroup,Lesson,Activity,Resource,Objective,Vocabulary
+        Example: runner.rb -l -u csd1-2021 -m Activity,Resource,Objective
+        Example: runner.rb -l -u csp2-2021,csp3-2021,csp4-2021 -m Lesson
```

Comment:
I just noticed that these all say runner.rb. could use `#{$0}` to get the right command name?